### ISSUE TITLE
Nvidia Shadowplay doesnt work properly

### ISSUE THREAD

**Issue by HessamoddinS** [nwjs/nw.js#5096]
Nvidia Shadowplay doesn't work anymore. When I record the gameplay of a game I have exported with NW.js (v0.14.5), the video is stuck in the first frame and the gameplay is not recorded. I didn't have this problem with (v0.13.2). Haven't checked if it works with the latest version yet.

**Comment by rogerwang** [comment]
Please check the latest NW version and the Chrome browser as well.

**Comment by rogerwang** [comment]
close until there is more information.

**Comment by HessamoddinS** [comment]
So I tested with both Chrome and NW (v0.15.0) [I'm using Construct 2 and v0.15.0 is the latest version they support]. The video shows the gameplay when they are not in fullscreen. However when they are in fullscreen i can't record (The hotkey for recording on Shadowplay is Alt+F9, when i press it, it doesnt work. Didn't have this problem with v0.13.2). So I started recording before i go to fullscreen, then went to fullscreen to see what the program records. In NW as i mentioned before, the video only shows the first frame when it is on fullscreen untill i exit fullscreen. In Chrome the video shows the first few seconds until the notification (Press F11 to exit fullscreen) fades away and then it freezes. Similar to NW untill I exit fullscreen. So I think this issue is related to Chromium. Shadowplay works with Chromium 49 but not 50 and 51 (In fullscreen)
